numba/tests/test_ufuncs.py

```diff
@@ -609,17 +614,33 @@ def test_fmin_ufunc(self, flags=no_pyobj_flags):
 
     ############################################################################
     # Floating functions
+
+    def bool_additional_inputs(self):
+        return [
+            (np.array([True, False], dtype=np.bool_),
+             types.Array(types.bool_, 1, 'C')),
+            ]
+
     @tag('important')
     def test_isfinite_ufunc(self, flags=no_pyobj_flags):
-        self.unary_ufunc_test(np.isfinite, flags=flags)
+        self.unary_ufunc_test(
```

Comment:
Should this also get a test for the `datetime` types?

Reply:
The datetime types are tested in `numba.tests.test_ufuncs.TestLoopTypesDatetimeNoPython`.
i.e. `python runtests.py "numba.tests.test_ufuncs.TestLoopTypesDatetimeNoPython.test_isfinite_M_?"`

Reply:
Great, thanks for checking.